composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "silverstripe/admin": "^1.13@dev",
+        "silverstripe/admin": "^1.13.99",
```

Comment:
Needs to be changed

Reply:
Done. See https://github.com/silverstripe/silverstripe-admin/releases/tag/1.13.20